UniversityDatabase/Employee.cpp

```diff
@@ -1 +1,17 @@
-#include "Employee.h"
\ No newline at end of file
+#include "Employee.h"
+
+Employee::Employee(std::string Name, std::string Surname, std::string Pesel, std::string Address, Gender Gender, float Earnings) :
+	Person(Name, Surname, Pesel, Address, Gender), earnings { Earnings }
+{
+	earnings = Earnings;
+}
+
+Employee::~Employee()
+{
+}
+
+void Employee::PrintPersonalInformation()
```

Comment:
In the Person and derived classes add the necessary getters.
Methods handling e.g. writing data on screen should be moved to Database class